docs/Installation.md

```diff
@@ -4,7 +4,7 @@ To install and use ML-Agents, you need install Unity, clone this repository and
 install Python with additional dependencies. Each of the subsections below
 overviews each step, in addition to a Docker set-up.
 
-## Install **Unity 2017.1** or Later
+## Install **Unity 2017.4.10f1** or Later
```

Comment:
Does it have to be 2017.4._10_ or later? Could it be any version of 2017.4?

Reply:
I think we should just say `2017.4`

Reply:
But we haven't tested on 2017.4's early versions, have we?